data-otservbr-global/scripts/actions/tools/rust_remover.lua

```diff
@@ -33,52 +21,91 @@ local config = {
 		{ { 96501, 99500 }, PALADIN_ARMOR },
 		{ { 99501, 100000 }, CROWN_ARMOR },
 	},
-	[8896] = { -- rare rusty armor
+	[8896] = { -- slightly rusted armor
 		{ { 1, 50000 } },
-		{ { 50001, 60000 }, CHAIN_ARMOR },
-		{ { 60001, 70000 }, SCALE_ARMOR },
-		{ { 70001, 80000 }, BRASS_ARMOR },
-		{ { 80001, 87500 }, PLATE_ARMOR },
-		{ { 87501, 93000 }, KNIGHT_ARMOR },
-		{ { 93001, 96000 }, PALADIN_ARMOR },
-		{ { 96000, 97000 }, CROWN_ARMOR },
-		{ { 96001, 98500 }, GOLDEN_ARMOR },
-		{ { 98501, 99500 }, DRAGON_SCALE_MAIL },
-		{ { 99501, 100000 }, MAGIC_PLATE_ARMOR },
+		{ { 50001, 60000 }, BRASS_ARMOR },
+		{ { 60001, 70000 }, PLATE_ARMOR },
+		{ { 70001, 80000 }, KNIGHT_ARMOR },
+		{ { 80001, 87500 }, CROWN_ARMOR },
+		{ { 87501, 93000 }, PALADIN_ARMOR },
 	},
-	[8897] = { -- common rusty legs
+	[8897] = { -- heavily rusted legs
 		{ { 1, 26764 } },
 		{ { 26765, 60000 }, STUDDED_LEGS },
 		{ { 60001, 85000 }, CHAIN_LEGS },
 		{ { 85001, 98000 }, BRASS_LEGS },
 		{ { 98001, 99500 }, PLATE_LEGS },
-		{ { 99501, 100000 }, KNIGHT_LEGS },
 	},
-	[8898] = { -- semi-rare rusty legs
+	[8898] = { -- rusted legs
 		{ { 1, 35165 } },
 		{ { 35166, 60000 }, STUDDED_LEGS },
 		{ { 60001, 75000 }, CHAIN_LEGS },
 		{ { 75001, 87500 }, BRASS_LEGS },
 		{ { 87501, 95500 }, PLATE_LEGS },
 		{ { 95501, 98250 }, KNIGHT_LEGS },
 		{ { 98251, 99250 }, CROWN_LEGS },
-		{ { 99251, 100000 }, GOLDEN_LEGS },
 	},
-	[8899] = { -- rare rusty legs
+	[8899] = { -- slightly rusted legs
 		{ { 1, 50000 } },
 		{ { 50001, 75000 }, BRASS_LEGS },
 		{ { 75001, 90000 }, PLATE_LEGS },
 		{ { 90001, 97500 }, KNIGHT_LEGS },
 		{ { 97501, 99000 }, CROWN_LEGS },
 		{ { 99001, 100000 }, GOLDEN_LEGS },
 	},
+	--[[	[8900] = { -- heavily rusted shield
```

Comment:
Qual o motivo de estar comentado?

Reply:
The item exists in the game but is not in use yet in global tibia i just put it there for the id and in case it gets put into global. 

https://tibia.fandom.com/wiki/Heavily_Rusted_Shield

Reply:
Ok, thanks for the explanation.